Index: templates/base.html.twig
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\n<html>\n    <head>\n        <meta charset=\"UTF-8\">\n        <title>{% block title %}Welcome!{% endblock %}</title>\n        <link rel=\"icon\" href=\"data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 128 128%22><text y=%221.2em%22 font-size=%2296%22>⚫️</text></svg>\">\n        {# Run `composer require symfony/webpack-encore-bundle` to start using Symfony UX #}\n        {% block stylesheets %}\n            {{ encore_entry_link_tags('app') }}\n        {% endblock %}\n\n        {% block javascripts %}\n            {{ encore_entry_script_tags('app') }}\n        {% endblock %}\n    </head>\n    <body>\n        {% block body %}oiuytrtyuio{% endblock %}\n    </body>\n</html>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/templates/base.html.twig b/templates/base.html.twig
--- a/templates/base.html.twig	(revision ae401a708f39f3aa15c8267894479842dc71e02a)
+++ b/templates/base.html.twig	(date 1669202276439)
@@ -14,6 +14,6 @@
         {% endblock %}
     </head>
     <body>
-        {% block body %}oiuytrtyuio{% endblock %}
+        {% block body %}oiuyuio{% endblock %}
     </body>
 </html>
